src/NosCore.GameObject/Providers/UpgradeService/IUpgradeService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using ChickenAPI.Packets.Enumerations;
+using NosCore.GameObject.Networking.ClientSession;
+using NosCore.GameObject.Providers.InventoryService;
+
+namespace NosCore.GameObject.Providers.UpgradeService
+{
+    public interface IUpgradeService
+    {
+        public void HandlePacket(UpgradePacketType type, ClientSession clientSession, InventoryItemInstance item1,
```

Comment:
This is not a good name for the sum upgrade

Reply:
will be removed